src/Controller/DefaultController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Controller;
+
+use App\Security\OpenIdConfigurationProvider;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class DefaultController extends AbstractController
+{
+    /**
+     * @Route("/", name="default")
+     */
+    public function index(): Response
+    {
+        return $this->render('default/index.html.twig', [
+            'controller_name' => 'DefaultController',
+        ]);
+    }
+
+    /**
+     * @Route("/login", name="login")
+     */
+    public function login(SessionInterface $session, array $openIdProviderOptions = []): Response
+    {
+        $provider = new OpenIdConfigurationProvider([
+            'scope' => 'openid',
+            'response_type' => 'id_token',
+            'response_mode' => 'query',
```

Comment:
These options will probably always be the same, so consider settings them as defaults in the `OpenIdConfigurationProvider` constructor, e.g.

```
  public function __construct(array $options) {
    parent::__construct($options + [
      'scope' => 'openid',
      'response_type' => 'id_token',
       'response_mode' => 'query',
    ]);
  }
```

This will also  make it easier to refactor the `OpenIdConfigurationProvider` into a reusable Symfony bundle at some point.